### PR TITLE
fix: ununifi/mint/cdps

### PR DESCRIPTION
**Pull request by taikiFurusyo**
This PR is for PR [#241](https://github.com/UnUniFi/utils/pull/241).

### CODE REVIEW

File: projects/portal/src/app/views/mint/cdps/cdps.component.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
   <div class="flex flex-row">
     <h3 matSubheader>CDPs</h3>
     <span class="flex-auto"></span>
-    <button mat-flat-button routerLink="/mint/cdps/create" color="accent">
+    <button mat-flat-button routerLink="/cdp/cdps/create" color="accent">
```
Comment (Senna46) [inline projects/portal/src/app/views/mint/cdps/cdps.component.html:5]: As you will see when you check the operation, I have changed it from /cdp/cdps to /mint/cdps.
It's about time they recognize it.

**Review by Senna46 — CHANGES_REQUESTED** [review]

File: projects/portal/src/app/views/mint/cdps/cdps.component.html (hunk 1 above)
Comment (taikiFurusyo) [reply projects/portal/src/app/views/mint/cdps/cdps.component.html:5]: Ohh, sorry.
I will check changed files by myself from now on.

Comment (taikiFurusyo) [reply projects/portal/src/app/views/mint/cdps/cdps.component.html:5]: Ohh, sorry.
I will check changed files by myself from now on.

**Review by Senna46 — APPROVED** [review]
LGTM

**Pull request merged**